src/D2L.Bmx/Okta/OktaApi.cs

```diff
@@ -38,7 +38,11 @@ internal class OktaApi : IOktaApi {
 	}
 
 	void IOktaApi.SetOrganization( string organization ) {
-		_httpClient.BaseAddress = new Uri( $"https://{organization}.okta.com/api/v1/" );
+		if( !organization.Contains( '.' ) ) {
```

Comment:
consider trimming trailing slashes, leading "http://" or leading "https://" to be more tolerant of various user inputs
e.g. "https://d2l.okta.com/"